client/interfaces.go

```diff
@@ -73,6 +73,7 @@ type InstanceServer interface {
 	ImageServer
 
 	// Server functions
+    GetMetadataConfiguration() (config *map[string]any, err error)
```

Comment:
Tabs should be used instead of spaces. Thx

Reply:
Or more precisely: go fmt should be used for all code when saving in your ide